scripts/update-search-index.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+node --max-old-space-size=8192 ./scripts/search/main.js "$1"
```

Comment:
would it make sense to just configure `--max-old-space-size` using the `NODE_OPTIONS` env variable in the workflow config file?

Reply:
Ah, forgot that was there. Yep! I’ll try that.